Decred Wallet/Features/Overview/Overview.storyboard

```diff
@@ -9,325 +9,326 @@
         <capability name="Safe area layout guides" minToolsVersion="9.0"/>
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
-    <customFonts key="customFonts">
-        <array key="SourceSansPro-Regular.ttf">
-            <string>SourceSansPro-Regular</string>
```

Comment:
This is the font used in the mockups. Is it what is in use now?

Reply:
yes. i am using the same fonts as the one in the mockups.